doc/references.bib

```diff
@@ -1191,6 +1201,33 @@ @inproceedings{Varoquaux2010b
 	series = {Lecture notes in computer science}
 }
 
+@article{Varoquaux2010c,
+	title = {A group model for stable multi-subject ICA on fMRI datasets},
+	author = {Varoquaux, Gael
+	and Sadaghiani, Sepideh
+	and Pinel, Philippe
+	and Kleinschmidt, Andreas
+	and Poline, Jean-Baptiste
+	and Thirion, Bertrand},
+	year = {2010},
+    doi={10.1016/j.neuroimage.2010.02.010},
```

Comment:
CI says that you are missing a journal in those refs:

https://github.com/nilearn/nilearn/actions/runs/4314727598/jobs/7528138270#step:33:609